doc/devel/contributing.rst

```diff
@@ -595,3 +595,43 @@ example code you can load it into a file handle with::
 
     import matplotlib.cbook as cbook
     fh = cbook.get_sample_data('mydata.dat')
+
+
+Baseline image generation
+-------------------------
+The first time this test is run, there will be no baseline image to compare against, so the test will fail. The baseline
```

Comment:
This is probably a good place to put the logic of how the images are going to be managed going forward (either pulled from a "blessed" package or generated in-place).  We should also add some thing about the logic of why we think that these sometimes re-generated images (!!) are going to still provide the quality assurance that we are getting from a static set of images.

Reply:
Same version of matplotlib baseline images package as that of the matplotlib.